pkg/plugins/platform/local.go

```diff
@@ -201,9 +201,23 @@ func (l *LocalApp) ImportDB(imPath string, extPath string) error {
 		}
 	}
 
-	err = l.Exec("db", true, "./import.sh")
+	matches, err := filepath.Glob(filepath.Join(dbPath, "*.sql"))
 	if err != nil {
-		return fmt.Errorf("failed to execute import: %v", err)
+		return err
+	}
+
+	if len(matches) < 1 {
```

Comment:
I don't think this is the right logic. Purge all if we found one or more sql files? I'd expect to purge all pre-emptively before un-archiving.